src/libraries/System.Management/src/System/Management/ManagementDateTime.cs

```diff
@@ -193,7 +193,7 @@ public static DateTime ToDateTime(string dmtfDate)
                 throw new ArgumentOutOfRangeException(nameof(dmtfDate));
             }
 
-
+            // codeql[cs/leap-year/unsafe-date-construction-from-two-elements] - DateTime not constructed from multiple elements - it's parsed from a string.
```

Comment:
```suggestion
            // codeql[cs/leap-year/unsafe-date-construction-from-two-elements] - DateTime not constructed from multiple elements - it's parsed from a string.  It would be intentional to throw if an invalid combination occurred.
```